core/commands/dht.go

```diff
@@ -581,3 +597,18 @@ PutValue will store the given key value pair in the dht.
 	},
 	Type: notif.QueryEvent{},
 }
+
+func escapeDhtKey(s string) (key.Key, error) {
+	if s[0] != '@' {
+		return key.B58KeyDecode(s), nil
+	}
```

Comment:
maybe it could just detect it from the path? e.g.

```sh
get <base58-hash> # assumed straight dht key.
get /pk/<base58-hash> # decode and so on.
```
also, it may be that, instead of `/pk/<raw-hash-bytes>` we should be doing `/pk/<base58-hash>` -- it may be easier on the user. the whole value is hashed again by the dht, so it doesn't matter there.